.travis.yml

```diff
@@ -3,14 +3,19 @@ language: julia
 os:
   - linux
   - osx
+  - windows
 julia:
   - 0.7
   - 1.0
+  - 1.1
```

Comment:
Let's not bother with 0.7 and 1.1, since they're not maintained?

Reply:
Keep 1.1,
drop 0.7

Reply:
> Keep 1.1, for now.

What's the reason for this? Seems as good a plan as any to drop support for versions at the same rate as the language.